config/kubernetes/sig-release/teams.yaml

```diff
@@ -175,32 +176,24 @@ teams:
   release-engineering:
     description: Members of the Release Engineering subproject, including Build Admins,
       Patch Release Team, Branch Managers, and Release Manager Associates.
-    maintainers:
-    - nikhita # Release Manager Associate
     members:
     - aleksandra-malinowska # subproject owner / Build Admin / Patch Release Team
     - alexeldeib # Release Manager Associate
-    - bubblemelon # Branch Manager
     - calebamiles # subproject owner
     - cpanato # Branch Manager
     - dougm # Patch Release Team
     - feiskyer # Patch Release Team
-    - girikuncoro # Release Manager Associate
     - hoegaarden # subproject owner / Patch Release Team
     - idealhack # Patch Release Team
-    - imkin # Release Manager Associate
```

Comment:
I had like to stay on this and contribute.

Reply:
Moving forward here. Discussed this w/ Dhawal offline.